internal/app/networkd/pkg/networkd/networkd.go

```diff
@@ -172,17 +172,17 @@ func (n *Networkd) Configure() (err error) {
 
 			// Ensure link exists
 			if err = netif.Create(); err != nil {
-				log.Println("error creating nic", err)
+				log.Printf("error creating nic: %v\n", err)
```

Comment:
nit, i don't think we want `\n` here

Reply:
We don't?

Reply:
https://play.golang.org/p/R5PHhPKwWI8

Reply:
Ahh, I guess that is not true with `log`: https://play.golang.org/p/k4dZ7GA3wnT

Reply:
Fixed.